src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1556,11 +1683,11 @@ private void doEncodeTransferFragments(
             int fragmentRemaining = fragmentLimit - fragmentProgress;
 
             AmqpTransferFW transferCont = amqpTransferRW
-                    .wrap(frameBuffer, FRAME_HEADER_SIZE, frameBuffer.capacity())
+                    .wrap(frameBuffer, FRAME_HEADER_SIZE + PERFORMATIVE_SIZE, frameBuffer.capacity())
                     .handle(handle)
                     .more(1)
                     .build();
-            int fragmentSizeCont = (int) encodeMaxFrameSize - FRAME_HEADER_SIZE - transferCont.sizeof();
+            int fragmentSizeCont = (int) encodeMaxFrameSize - FRAME_HEADER_SIZE - PERFORMATIVE_SIZE - transferCont.sizeof();
```

Comment:
Given that `replySharedBudget` is an `int`, let's change the type of `encodeMaxFrameSize` to `int` and eliminate these casts.